### PR TITLE
#avoid saving css data into plugin folder

### PR DESCRIPTION
**Pull request by cpsychogyios**

### CODE REVIEW

File: admin/admin.php [hunk 1]
```diff
@@ -150,16 +150,6 @@ function tlms_resyncCourse() {
 }
 add_action('wp_ajax_tlms_resynch', 'tlms_resyncCourse');
 
-function cssPage() {
```
Comment (epolikrati) [inline admin/admin.php:153]: This file will no longer exists in our plugin structure and its logic has been transferred to Admin.php.

File: src/Enqueue.php [hunk 2]
```diff
@@ -5,22 +5,25 @@
 
 namespace TalentlmsIntegration;
 
+use TalentlmsIntegration\Pages\Admin;
+
 class Enqueue {
 
 	public function register(){
 		add_action('admin_enqueue_scripts', array($this, 'tlms_enqueueAdminScripts'));
+		add_action('tlms_enqueue_styles', array($this, 'tlms_enqueueStyles'));
```
Comment (epolikrati) [inline src/Enqueue.php]:
```suggestion
		add_action('wp_enqueue_scripts', array($this, 'tlms_enqueueStyles'));
```

File: src/Pages/Admin.php [hunk 3]
```diff
@@ -89,9 +91,58 @@ public function tlms_integrationsPage () {
...
+			$customCssFileContent = $wp_filesystem->get_contents($customCssFileName);
+		}
+
+		$presentCssFileName = pathinfo($customCssFileName)['filename'];
+		require_once TLMS_BASEPATH.'/templates/css.php';
+	}
+
+	public static function getCustomCssFilePath(): string{
+		$upload_dir = wp_upload_dir();
+		$dir = trailingslashit($upload_dir['basedir']).TLMS_UPLOAD_DIR;
```
Comment (epolikrati) [inline src/Pages/Admin.php]: We are not loading the correct recourse path for our custom css.

File: talentlms.php [hunk 4]
```diff
@@ -1,66 +1,66 @@
...
+	require_once plugin_dir_path( __FILE__ ) . 'vendor/autoload.php';
+}
+
+/**
+ * Define constants
+ */
+define('TLMS_BASEPATH', dirname(__FILE__));
+define('TLMS_BASEURL', plugin_dir_url(__FILE__));
+define('TLMS_VERSION', '6.6.9.5');
+define('TLMS_UPLOAD_DIR', 'talentlmswpplugin');
```
Comment (epolikrati) [inline talentlms.php:25]: We should provide the correct upload dir file here.

**Review by epolikrati — CHANGES_REQUESTED** [review]

File: admin/admin.php (hunk 1 above)
Comment (cpsychogyios) [reply admin/admin.php:153]: We agreed that all old files will be removed at the end of refactoring at the base branch in order to keep record of the changes.

File: src/Enqueue.php (hunk 2 above)
Comment (cpsychogyios) [reply src/Enqueue.php]: Fixed [3862538](https://github.com/epignosis/talentlms-wordpress/pull/21/commits/38625388d40063bac9ed349eb1916cd90ccbb0a2)

File: src/Pages/Admin.php (hunk 3 above)
Comment (cpsychogyios) [reply src/Pages/Admin.php]: Fixed [3862538](https://github.com/epignosis/talentlms-wordpress/pull/21/commits/38625388d40063bac9ed349eb1916cd90ccbb0a2)

File: talentlms.php (hunk 4 above)
Comment (cpsychogyios) [reply talentlms.php:25]: Sorry, I didn't catch what do you mean..

Comment (cpsychogyios) [reply talentlms.php:25]: Fixed it [0ef8c01](https://github.com/epignosis/talentlms-wordpress/pull/21/commits/0ef8c01f6d95b7a926a96ea19e24ce8c29e2e2f5)

File: admin/admin.php (hunk 1 above)
Comment (epolikrati) [reply admin/admin.php:153]: So we don't need to interfere in these files.

**Comment by cpsychogyios** [comment]
@epolikrati proposed changes reverted, after our discussion over slack, for performance issues here [92eeaa4](https://github.com/epignosis/talentlms-wordpress/pull/21/commits/92eeaa4d75d2c0355ef5e017d0194ce85aaa6e10)

**Review by epolikrati — APPROVED** [review]

**Pull request merged**